src/zenhub.py

```diff
@@ -152,16 +77,11 @@ def __init__(self, repo: 'ZenHubRepo', key: str = None, content: dict = None):
         """
 
         super().__init__()
-
         self.repo = repo
 
-        if key and repo:
-            response = requests.get(f'{self.repo.url}{self.repo.id}/issues/{key}', headers=self.repo.headers)
-            if response.status_code == 200:
-                content = response.json()
-                content['issue_number'] = key
-            else:
-                raise ValueError(f'{response.status_code} Error: {response.text}')
+        if not content:
+            content = self.repo.api_call(requests.get, f'{self.repo.id}/issues/{key}')
+            content['issue_number'] = key
```

Comment:
What happens if `key = None` and `content = None`? Don't you need either one or the other?

Reply:
Added a condition to check for this case and raise an error